src/main/resources/public/js/app.js

```diff
@@ -190,6 +185,7 @@ function scanBlinkAnimation(row, column, color, repeat) {
 function displayGameState(gameModel){
 $( '#MyBoard td'  ).css("background-color", "DarkBlue");
 $( '#TheirBoard td'  ).css("background-color", "DarkBlue");
+var audio = new Audio('audio/0477.mp3');
```

Comment:
Everyone knows a good User Interface interacts with as many human senses as possible! Hearing feedback for hitting a civilian ship is just as important as seeing it as a different color!